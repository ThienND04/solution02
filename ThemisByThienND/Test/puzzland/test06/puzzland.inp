7
4 4
S I N G 
N G
S N
I G
S G
6 10
W X H B D F 
B F
W F
H F
W X
X F
H D
D F
H B
B D
X D
4 6
U O H D 
U O
O D
O H
H D
U D
U H
6 14
N B W E I D 
B I
I D
E D
W E
B D
B W
N I
N D
W I
N W
B E
W D
N B
N E
9 34
C W F H D S K B M 
F K
W K
F H
H K
B M
F D
C F
W D
W F
C B
H S
W S
D K
S B
H M
C W
K B
D B
W M
C S
S M
W H
C H
W B
D S
C M
F S
F B
K M
F M
S K
H B
C K
D M
8 23
L Q Y D H B P S 
L Y
Y B
Y P
H P
L B
Q H
L Q
L H
L P
Q B
D P
H S
Q P
H B
Y S
D H
P S
Q D
B S
Q Y
L D
Y D
B P
9 32
U R X O N L M H T 
U R
N H
U H
X M
X N
X H
R X
O L
U L
H T
O T
O M
R M
R N
M H
R L
O H
L T
N T
O N
L M
R T
U T
N M
N L
X T
X L
L H
R O
X O
U N
M T
