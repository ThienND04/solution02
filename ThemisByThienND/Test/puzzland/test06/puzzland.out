IMPOSSIBLE
WXDBHF
UHOD
NBEWID
CBDFHKSWM
LBHDPQYS
UHLMNORXT
